homeassistant/components/traccar_server/coordinator.py

```diff
@@ -46,25 +54,24 @@ class TraccarServerCoordinator(DataUpdateCoordinator[TraccarServerCoordinatorDat
     def __init__(
         self,
         hass: HomeAssistant,
+        config_entry: ConfigEntry,
         client: ApiClient,
-        *,
-        events: list[str],
-        max_accuracy: float,
-        skip_accuracy_filter_for: list[str],
-        custom_attributes: list[str],
```

Comment:
Why was this changed? This change is not reflected in the title or description of this PR.

Reply:
I thought it wouldn't make sense to pass in all these single config entry data parameters, when we already pass in the config entry itself.

Reply:
By doing this, you are moving the responsibility of extracting/validating this data from the setup (where it is expected to find it) to the coordinator.